src/data/faq_de.json

```diff
@@ -874,7 +874,7 @@
                                 "active": false,
                                 "textblock": [
                                     "Im Allgemeinen ist der digitale Impfnachweis  (beziehungsweise digitale Test- oder Genesenennachweis) per Corona-Warn-App lediglich ein vollkommen freiwilliges und ergänzendes Angebot. Wenn Sie keinen digitalen Impfnachweis besitzen oder diesen verloren haben, ist der Impfnachweis über das bekannte „gelbe Heft“ (gelber Impfausweis) in der Regel weiterhin möglich und gültig. Außerdem können Sie Corona-Impfungen auch mit dem ausgedruckten Corona-Impfzertifikat für die vollständige Impfung nachweisen. Digitale Zertifikate können nachträglich durch Apotheken ausgestellt werden.",
-                                    "<b>Wichtig:</b> Prüfen Sie bitte die lokalen Verordnungen der Bundesländer."
+                                    "<b>Wichtig:</b> Prüfen Sie bitte die lokalen Verordnungen der Bundesländer. Berlin beispielsweise verlangt, dass der Nachweis der Impfung gegen oder Genesung von SARS-CoV2 digital verifizierbar sein muss (Siehe <a href='https://www.berlin.de/corona/massnahmen/verordnung/' target='_blank' rel='noopener noreferrer'>https://www.berlin.de/corona/massnahmen/verordnung/</a> §&nbsp;8a Absatz&nbsp;2 Ziffer&nbsp;4). Das heißt, dass Sie gegebenenfalls ein entsprechendes digitales Zertifikat der EU entweder in der Corona-Warn-App, der CovPass-App oder als Papierausdruck zur Überprüfung vorzulegen haben."

```

Comment:
The reference to Berlin has nothing to do with backups and reverts the article to the version before PR https://github.com/corona-warn-app/cwa-website/pull/3232 was merged.
This (and other unrelated changes) should not be part of this branch / PR.